### ISSUE TITLE
`useSeoMeta()`: no longer allow null value

### ISSUE THREAD

**Issue by paraboul** [unjs/unhead#226]
### Environment

`@unhead/vue@1.6.2`

### Reproduction

-

### Describe the bug

`useSeoMeta` used to allow `null` value to explicitly discard a specific property.

**Example** :
In this case, relying on the potentially `undefined` value would emit a `meta description` tag with no value :
```js
useSeoMeta({
    description: pageContent.value?.meta?.meta_description,
})
```
**Output** :
```html
<meta name="description" content>
```

While relying on `null` would simply emit no tag at all :
```js
useSeoMeta({
    description: pageContent.value?.meta?.meta_description || null,
})
```

A recent change (in 1.6 I guess) broke this and no longer allows null value (triggering an exception in Vue) :

```
index.mjs:364 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at flattenMetaObjects (index.mjs:364:37)
    at flattenMetaObjects (index.mjs:376:26)
    at unpackMeta (index.mjs:404:8)
    at index.mjs:67:13
    at callWithErrorHandling (runtime-core.esm-bundler.js:158:18)
    at callWithAsyncErrorHandling (runtime-core.esm-bundler.js:166:17)
    at ReactiveEffect.getter [as fn] (runtime-core.esm-bundler.js:1702:16)
    at ReactiveEffect.run (reactivity.esm-bundler.js:178:19)
    at doWatch (runtime-core.esm-bundler.js:1797:12)
```

There is currently no simple syntax to conditionally emit a specific tag and my two cents would be to simply rely on `undefined` as I don't see any specific need for an empty tag.

Thanks 😊

**Comment by harlan-zw** [comment]
@dargmuesli Seems to be from the meta changes you did

**Comment by harlan-zw** [comment]
Have pushed up a fix but working on a few other things so will release when I can

**Comment by paraboul** [comment]
@harlan-zw 

The commit fixed the crash but `null` no longer remove the tag in some cases.
```
useSeoMeta({
    description: null,
    ogImage: null,
})
```

This still emit the `og:Image` (but not the description) :

**full output** :
```
<meta property="og:image">
```

**Comment by harlan-zw** [comment]
Thanks for flagging that, have pushed up a fix in 1.7.2
